docs/user-manual/modules/ROOT/pages/index.adoc

```diff
@@ -14,6 +14,8 @@ routing rules in your IDE, whether in a Java or XML editor.
 
 For a deeper and better understanding of Apache Camel, an xref:faq/what-is-camel.adoc[Introduction] is provided.
 
+Furthermore, camel has added a new API i.e. Endpoint-dsl. More information can be found under documentation section.
```

Comment:
Looking at this better, I don't think this is the right place for this. But we could move it later